client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/boost-site-performance.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from '@automattic/components';
+import { Icon, help } from '@wordpress/icons';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { jetpackBoostDesktopIcon, jetpackBoostMobileIcon } from '../../icons';
+import { getBoostRating, getBoostRatingClass } from '../utils';
+import ExpandedCard from './expanded-card';
+import type { BoostData } from '../types';
+
+interface Props {
+	boostData: BoostData;
+	hasBoost: boolean;
+}
+
+export default function BoostSitePerformance( { boostData, hasBoost }: Props ) {
+	const translate = useTranslate();
+
+	const { overall: overallScore, mobile: mobileScore, desktop: desktopScore } = boostData;
+
+	const components = {
+		strong: <strong></strong>,
+	};
+
+	return (
+		<ExpandedCard
+			header={ translate( 'Boost site performance' ) }
+			isEnabled={ !! hasBoost }
+			emptyContent={ translate(
+				'Add {{strong}}Boost{{/strong}} to see your site performance scores',
+				{
+					components,
+				}
+			) }
+		>
+			<div className="site-expanded-content__card-content-container site-expanded-content__boost">
+				<div className="site-expanded-content__card-content">
+					<div className="site-expanded-content__card-content-column">
+						<div
+							className={ classNames(
+								'site-expanded-content__card-content-score',
+								getBoostRatingClass( overallScore )
+							) }
+						>
+							{ getBoostRating( overallScore ) }
+							<Icon size={ 24 } className="site-expanded-content__help-icon" icon={ help } />
```

Comment:
Size 24 looks a little large per the mock up. It shows it should be 12px by 12px

Reply:
Changed the size to 20 now. 18 looks too small.

Reply:
Also, reduced size of device icons

<img width="350" alt="Screenshot 2023-03-16 at 8 25 46 PM" src="https://user-images.githubusercontent.com/10586875/225656895-d4a0300b-b890-4004-b558-38a36bb00029.png">